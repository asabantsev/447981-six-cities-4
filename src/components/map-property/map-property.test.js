import React from "react";
import renderer from "react-test-renderer";
import MapProperty from "./map-property.jsx";

const OFFERS = [
  {
    coords: [52.3909553943508, 4.85309666406198],
  },
  {
    coords: [52.369553943508, 4.85309666406198],
  },
  {
    coords: [52.3909553943508, 4.929309666406198],
  },
  {
    coords: [52.3809553943508, 4.939309666406198],
  },
];

describe(`MapSnapTest`, () => {
  it(`Map should render map component with pins`, () => {
    const tree = renderer
      .create(<MapProperty
        offers = {OFFERS.slice(-3)}
        currentOffer = {OFFERS[0]}
      />,
      {
        createNodeMock: () => document.createElement(`div`)
      }
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
