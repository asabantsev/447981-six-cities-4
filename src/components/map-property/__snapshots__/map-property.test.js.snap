// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapSnapTest Map should render map component with pins 1`] = `
<section
  className="property__map map"
/>
`;
